.sourcery.yaml

```diff
@@ -0,0 +1,170 @@
+    - merge-list-extend
+    - merge-nested-ifs
+    - merge-repeated-ifs
+    - merge-set-add
+    - min-max-identity
+    - missing-dict-items
+    - move-assign
+    - move-assign-in-block
+    - non-equal-comparison
+    - path-read
+    - raise-from-previous-error
+    - reintroduce-else
+    - remove-assert-true
+    - remove-dict-items
+    - remove-none-from-default-get
+    - remove-pass-body
+    - remove-pass-elif
+    - remove-redundant-boolean
+    - remove-redundant-condition
+    - remove-redundant-constructor-in-dict-union
+    - remove-redundant-continue
+    # - remove-redundant-except-handler (without autofix)
+    # - remove-redundant-exception (without autofix)
+    - remove-redundant-if
+    - remove-redundant-pass
+    - remove-redundant-slice-index
+    - remove-str-from-fstring
+    - remove-str-from-print
+    - remove-unit-step-from-range
+    - remove-unnecessary-cast
+    - remove-unnecessary-else
+    # - remove-unreachable-code (without autofix)
+    - remove-unused-enumerate
+    - remove-zero-from-range
+    - replace-apply-with-method-call
+    - replace-dict-items-with-values
+    - replace-interpolation-with-fstring
+    - simplify-constant-sum
+    - simplify-dictionary-update
+    - simplify-division
+    - simplify-fstring-formatting
+    - remove-redundant-slice-index
+    - remove-str-from-fstring
+    - remove-str-from-print
+    - remove-unit-step-from-range
+    - remove-unnecessary-cast
+    - remove-unused-enumerate
+    - remove-zero-from-range
+    - replace-apply-with-method-call
+    - replace-apply-with-numpy-operation
+    - replace-dict-items-with-values
+    - set-comprehension
+    - simplify-boolean-comparison
+    - simplify-constant-sum
+    - simplify-dictionary-update
+    - simplify-generator
+    - simplify-len-comparison
+    - simplify-negative-index
+    - simplify-string-len-comparison
+    - simplify-substring-search
+    - skip-sorted-list-construction
+    - square-identity
+    - str-prefix-suffix
+    - sum-comprehension
+    - swap-nested-ifs
+    - swap-variable
+    # - switch (remove sim114)
+    - ternary-to-if-expression
+    - tuple-literal
+    - unwrap-iterable-construction
+    - use-any
+    - use-assigned-variable
+    - use-count
+    - use-datetime-now-not-today
+    - use-dict-items
+    - use-dictionary-union
+    - use-file-iterator
+    - use-fstring-for-concatenation
+    - use-fstring-for-formatting
+    - use-getitem-for-re-match-groups
+    - use-isna
+    - use-itertools-product
+    - use-join
+    - use-len
+    - use-named-expression
+    - use-string-remove-affix
+    - while-guard-to-condition
+    - while-to-for
+
+disable: []
+rule_types:
+  - refactoring
+  - suggestion
+  - comment
+
+python_version: "3.7"
+
+rules: []
+
+metrics:
+  quality_threshold: 25.0
+
+github:
+  labels: []
+  ignore_labels:
+    - sourcery-ignore
+  request_review: author
+  sourcery_branch: sourcery/{base_branch}
+
+clone_detection:
```

Comment:
i don't think it's wise for content, as we have a lot of dups, that can't be shared via api module.